config.yml

```diff
@@ -496,12 +496,20 @@ nodes:
       - name: flags
         type: flags
         kind: ArrayNodeFlags
+        comment: |
+          Represents whether or not there is a splat operator in the array.
       - name: elements
         type: node[]
+        comment: |
+          Represent list of elements in the array.
       - name: opening_loc
         type: location?
+        comment: |
+          Represents optional source location for opening token.
       - name: closing_loc
         type: location?
+        comment: |
+          Represents optional source location for closing token.
```

Comment:
Same comment as above for this. We want to know more than just rephrasing the name and type.